checks.d/zk.py

```diff
@@ -71,22 +84,21 @@ def check(self, instance):
         self.service_check('zookeeper.ruok', status, message=message,
                            tags=sc_tags)
 
-        # Read metrics from the `stat` output.
         try:
-            stat_out = self._send_command('stat', *cx_args)
+            mntr_out = self._send_command('mntr', *cx_args)
         except ZKConnectionFailure:
             self.increment('zookeeper.timeouts')
             raise
         else:
-            # Parse the response
-            metrics, new_tags, mode = self.parse_stat(stat_out)
-
-            # Write the data
-            for metric, value in metrics:
-                self.gauge(metric, value, tags=tags + new_tags)
+            metrics, state = self.parse_mntr(mntr_out)
+            state_tag = "mode:%s" % state
+            if state != 'inactive':
+                for name in metrics:
+                    self.gauge(name, metrics[name], tags=tags + [state_tag])
+            self.set('zookeeper.instances', state_tag, tags=[state_tag])
```

Comment:
What should be used as a unique value? We could provide the hostname in the config file but it doesn't have it right now because it only accesses localhost... or am I miss understanding something.